modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -608,10 +625,22 @@ SET STATUS SUSPENDED
 SET HOME DATABASE anotherDb
 ----
 
+The equivalent command using the xref:authentication-authorization/auth-objects.adoc[Auth Objects] syntax would be:
```

Comment:
should this one also have the `[role=label--new-5.24]`?

Reply:
yes, thanks. Added.